### PR TITLE
update how to config initial-commit-ts on tiup v1.9.2

### PR DESCRIPTION
**Pull request by nexustar**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@nexustar Please specify the applicable version in the PR description part and sign the  Contributor License Agreement. Thanks!

**Review by qqqdan — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -350,16 +350,18 @@ pump_servers:
...
 
+其中 `commit_ts` 字段自 tiup cluster v1.9.2 开始已经废弃，不再被记录到 drainer 的启动脚本中，需要将其配置为非默认值的用户，请参照下例在 `config` 中配置 `initial-commit-ts` 字段。
+
 `drainer_servers` 配置示例：
 
 ```yaml
 drainer_servers:
   - host: 10.0.1.21
     config:
+      initial-commit-ts: -1
```
Comment (shichun-0415) [inline tiup/tiup-cluster-topology-reference.md:364]: @nexustar L339 说它的默认值是 -1，这里是配置为非默认值的示例，-1 是不是不对呢？

**Review by AstroProfundis — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md (hunk 1 above)
Comment (nexustar) [reply tiup/tiup-cluster-topology-reference.md:364]: 没有说是非默认值的配置吧，当不配置这一项时是等同配置为-1的

Comment (shichun-0415) [reply tiup/tiup-cluster-topology-reference.md:364]: L356：需要将其配置为非默认值的用户，请参照下例在 `config` 中配置 `initial-commit-ts` 字段。

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -350,16 +350,18 @@ pump_servers:
 - `deploy_dir`
 - `data_dir`
 - `log_dir`
-- `commit_ts`
 - `arch`
 - `os`
 
+其中 `commit_ts` 字段自 tiup cluster v1.9.2 开始已经废弃，不再被记录到 drainer 的启动脚本中，需要将其配置为非默认值的用户，请参照下例在 `config` 中配置 `initial-commit-ts` 字段。
```
Comment (shichun-0415) [inline tiup/tiup-cluster-topology-reference.md:356]:
```suggestion
其中 `commit_ts` 字段自 tiup cluster v1.9.2 开始已经废弃，不再被记录到 drainer 的启动脚本中。如需要配置该字段，请参照下例在 `config` 中配置 `initial-commit-ts` 字段。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 3]
```diff
@@ -350,16 +350,18 @@ pump_servers:
 - `deploy_dir`
 - `data_dir`
 - `log_dir`
-- `commit_ts`
 - `arch`
 - `os`
 
+其中 `commit_ts` 字段自 tiup cluster v1.9.2 开始已经废弃，不再被记录到 drainer 的启动脚本中.仍需要配置该参数的用户，请参照下面的示例在 `config` 中配置 `initial-commit-ts` 字段。
```
Comment (shichun-0415) [inline tiup/tiup-cluster-topology-reference.md:356]:
```suggestion
其中 `commit_ts` 字段自 tiup cluster v1.9.2 开始已经废弃，不再被记录到 drainer 的启动脚本中。如果你仍需要配置该参数，请参照下面的示例在 `config` 中配置 `initial-commit-ts` 字段。
```

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**